### PR TITLE
Better scalable visualization of known mutations

### PR DESCRIPTION
**Pull request by joswinter**
Added two settings:
- the radius of the bookmarks
- the opacity of the radius

### CODE REVIEW

File: lifetiles-graph/src/main/java/nl/tudelft/lifetiles/graph/view/Bookmark.java [hunk 1]
```diff
@@ -38,10 +44,10 @@
     public Bookmark(final VertexView vertex,
             final KnownMutation knownMutation, final long segmentPosition,
             final double scale) {
-        super(vertex.HORIZONTALSCALE * scale, vertex.getHeight());
-        setLayoutX(vertex.getLayoutX() + segmentPosition
+        super(RADIUS);
+        setLayoutX(vertex.getLayoutX() + (segmentPosition + 0.5)
```
Comment (jorenham) [inline lifetiles-graph/src/main/java/nl/tudelft/lifetiles/graph/view/Bookmark.java]: magic number

File: lifetiles-graph/src/main/java/nl/tudelft/lifetiles/graph/view/Bookmark.java [hunk 2]
```diff
@@ -11,12 +12,12 @@
...
  *
  */
-public class Bookmark extends Rectangle {
+public class Bookmark extends Circle {
 
     /**
      * The standard opacity of the bookmark overlay.
      */
-    private static final double OPACITY = 0.5;
+    private static final double OPACITY = Double.parseDouble(Settings.get("bookmark_opacity"));
```
Comment (rutgervandenberg) [inline lifetiles-graph/src/main/java/nl/tudelft/lifetiles/graph/view/Bookmark.java]: Should make a final string for the settings property.

File: lifetiles-graph/src/main/java/nl/tudelft/lifetiles/graph/view/Bookmark.java [hunk 3]
```diff
@@ -24,6 +25,16 @@
     private static final Color STANDARD_COLOR = Color.YELLOW;
 
     /**
+     * The radius of the bookmark circle.
+     */
+    private static final double RADIUS = Double.parseDouble(Settings.get("bookmark_radius"));
```
Comment (rutgervandenberg) [inline lifetiles-graph/src/main/java/nl/tudelft/lifetiles/graph/view/Bookmark.java]: Should make a final string for the settings property.

File: lifetiles-graph/src/test/java/nl/tudelft/lifetiles/graph/view/TileViewTest.java [hunk 4]
```diff
@@ -157,7 +159,7 @@ public void drawAnnotationTest() throws IOException, InterruptedException {
         });
 
         Thread.sleep(1000);
-        assertEquals(rec.getHeight(), vView1.getHeight(), 1e-10);
+        assertEquals(bookmark.getRadius(), Double.parseDouble(Settings.get("bookmark_radius")), 1e-10);
```
Comment (rutgervandenberg) [inline lifetiles-graph/src/test/java/nl/tudelft/lifetiles/graph/view/TileViewTest.java]: Should make a final string for the settings property.

File: lifetiles-graph/src/test/java/nl/tudelft/lifetiles/graph/view/TileViewTest.java [hunk 5]
```diff
@@ -107,7 +109,7 @@ public void drawVerticeTest() {
 
     }
 
-    private Rectangle rec;
+    private Circle bookmark;
     private VertexView vView1;
```
Comment (albert-smit) [inline lifetiles-graph/src/test/java/nl/tudelft/lifetiles/graph/view/TileViewTest.java]: why not vertexView1?

**Comment by albert-smit** [comment]
looks good when the mentioned issues are fixed

**Comment by alangerak** [comment]
Looks good if issues are fixed.

**Pull request merged**